### PR TITLE
PLUGIN-1090 duplicate call to table check, timeout

### PR DESCRIPTION
**Pull request by sumitnigamcs**
duplicate call to spanner table check, timeout issue

### CODE REVIEW

**Comment by sumitnigamcs** [comment]
@rmstar @itsankit-google Please review

**Comment by sumitnigamcs** [comment]
@rmstar @itsankit-google Please review

**Review by rmstar — APPROVED** [review]

File: src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java [hunk 1]
```diff
@@ -153,10 +152,8 @@ public void prepareRun(BatchSinkContext context) throws Exception {
     }
   }
 
-  private void createTableIfNotPresent(DatabaseClient dbClient, Database database,
+  private void createTableIfNotPresent(Database database,
                                        Schema schema) throws ExecutionException, InterruptedException {
-    boolean tableExists = isTablePresent(dbClient);
-    if (!tableExists) {
       if (Strings.isNullOrEmpty(config.getKeys())) {
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java]: if condition has been removed, so I think we'll need to fix the indent

Comment (itsankit-google) [reply src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java]: @rmstar Should we also rename the method to `createTable`? As it no longer checks for existence of table.

Comment (rmstar) [reply src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java]: Yeah sg 👍

Comment (sumitnigamcs) [reply src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java]: indent corrected, method renamed

**Comment by sumitnigamcs** [comment]
@itsankit-google review comments addressed. Please see if we can merge this

**Review by itsankit-google — APPROVED** [review]
LGTM. Please raise a cherry-pick as well for 6.7 on `release/0.20` branch.

**Pull request merged**